aptos-move/framework/aptos-experimental/sources/large_packages.move

```diff
@@ -1,14 +1,48 @@
-/// This provides a framework for uploading large packages to standard accounts or objects.
+/// # Aptos Large Packages Framework
```

Comment:
I thought we want to put this into framework proper so everybody can use it?

Reply:
I am not sure if we settled on implementation and API and we are good to put it in the framework?

aptos-experimental is deployed everywhere where we start network from scratch, and on all other places - `testnet` and `mainnet` - we have it deployed already. so this change does allow every network to have it.

But if we are ready to put it in the framework, I can move it there.